### DISCUSSION TITLE
iPhone localhost testing

### DISCUSSION THREAD

**Discussion by tinleym** [vercel/next.js#47266] [Help]
### Summary

I'm having some issues testing my Next app on my iPhone. I'm using rewrites to get around localhost CORS issues, and everything works great on my dev machine.

On a connected iPhone, however, I'm only able to load routes. If I try to make a fetch from the client, the Safari Web Inspector is spits out the errors "Could not connect to the server" and "Fetch api cannot load [fetch url] due to access control checks". In the network > security tab, I'm getting "The resource was request insecurely".

Does anyone know the cause of this, and/or better yet, what to do?

**Comment by GNITOAHC** [comment]
Check if your ip is set to your host machine instead on localhost cause locahost iPhone won't link to your dev machine.

**Comment by noahlst** [comment]
Next.js 15.1 and later has a network (remote) URL on both `next dev` and `next start`. If you open the URL on the iPhone, it will come up as if it is `localhost`. The Safari remote web inspector works here too.
